packages/e2e-tests/config/setup-playwright.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	activateTheme,
+	deactivatePlugin,
+	enablePageDialogAccept,
+	isOfflineMode,
+	setBrowserViewport,
+	switchUserToAdmin,
+	switchUserToTest,
+	visitAdminPage,
+} from '@wordpress/e2e-test-utils';
+import { addQueryArgs } from '@wordpress/url';
+
+// The Jest timeout is increased because these tests are a bit slow
+jest.setTimeout( 100000 );
+
+/**
+ * Set of console logging types observed to protect against unexpected yet
+ * handled (i.e. not catastrophic) errors or warnings. Each key corresponds
+ * to the Puppeteer ConsoleMessage type, its value the corresponding function
```

Comment:
Did you mean "Playwright" in this line? Or does  I just got a little confused reading through this!

Reply:
This file is basically a copy-paste of the [setup-test-framework.js](https://github.com/WordPress/gutenberg/blob/5c5d2d26cc9a8b5134eb928c2d1a4339931a5703/packages/e2e-tests/config/setup-test-framework.js) with just a few alterations to kick-off Playwright (cherry-picked from https://github.com/WordPress/gutenberg/pull/23238). I should have updated those comments, sorry about the confusion!

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/34089/commits/d4e0654d9c8c61514022cfa7080428883ab4e740